languages/javalang/builder-renderer/commercetools-java-client/build.gradle

```diff
@@ -18,4 +18,8 @@ publishing {
 }
 
 dependencies {
+    compile project(':languages:javalang:builder-renderer:commercetools-java-base')
+    testCompile group: 'junit', name: 'junit', version: '4.12'
```

Comment:
We should use a consistent way of providing the coordinates of maven artifacts and change this to: `junit:junit:4.12`.